### PR TITLE
Module3 2

### PR DESCRIPTION
**Pull request by godoflame**
My end of Week 3 Code Review Pull Request.  Contains the 2 feature specs with capybara, support files, and changes to the user model/test spec and views for video#show and queue_items#index.

### CODE REVIEW

File: app/views/videos/show.html.haml [hunk 1]
```diff
@@ -10,7 +10,7 @@
         %p= @video.description
         .actions
           %a.btn.btn-primary(href="") Watch Now
-          = link_to "+ My Queue", queue_items_path(video_id: @video.id), method: 'POST', class: 'btn btn-default'
+          = link_to "+ My Queue", queue_items_path(video_id: @video.id), method: 'POST', class: 'btn btn-default' unless current_user.queued_video?(@video)
```
Comment (brandoncc) [inline app/views/videos/show.html.haml]: Since this line is so long, I would move the `unless` to its own line, instead of being inline.

Comment (godoflame) [inline app/views/videos/show.html.haml]: Yeah, I'll fix it in my next update, I keep forgetting with how wide I have my sublime text window that the line's getting kind of long.

Comment (brandoncc) [inline app/views/videos/show.html.haml]: You should setup the gutter at 80 characters to remind you.

Comment (godoflame) [inline app/views/videos/show.html.haml]: Wasn't even aware I could do that.  Will see if I can figure out how lol...

Comment (brandoncc) [inline app/views/videos/show.html.haml]: http://stackoverflow.com/questions/25900954/80-characters-right-margin-line-in-sublime-text-3

I should have said "ruler", not "gutter".

**Pull request merged**